operator/k8s_node.go

```diff
@@ -162,7 +161,7 @@ func runCNPNodeStatusGC(name string, clusterwide bool, ciliumNodeStore *store.Sh
 		controller.ControllerParams{
 			RunInterval: operatorOption.Config.CNPNodeStatusGCInterval,
 			DoFunc: func(ctx context.Context) error {
-				lastRun := time.Now().Add(-operatorOption.Config.NodesGCInterval)
+				lastRun := cilium_v2.NewTimestamp().Add(-operatorOption.Config.NodesGCInterval)
```

Comment:
This could be done in a different commit, like the one where you replaced the timestamp with metav1.Time

Reply:
Fixed